src/home/fetch-github/fetch-issues-preview.ts

```diff
@@ -34,12 +52,15 @@ export async function fetchIssuePreviews(): Promise<TaskNoFull[]> {
 
   return tasks;
 }
-function automaticLogin(error: unknown) {
+
+function rateLimitModal(error: unknown, message?: string) {
   const resetTime = error.response.headers["x-ratelimit-reset"];
   const resetParsed = new Date(resetTime * 1000).toLocaleTimeString();
 
   displayPopupMessage(
     `GitHub API rate limit exceeded.`,
-    `You have been rate limited. Please log in to GitHub to increase your GitHub API limits, otherwise you can try again at ${resetParsed}.`
+    !message
```

Comment:
Can you use normal if/else? Ternary is less readable.

Reply:
removed it completely with just passing in the string and reset time